portal/templates/redesign/teach_new/onboarding_print.html

```diff
@@ -8,15 +8,34 @@
 
 {% block scripts %}
 {{ block.super }}
+
+<script type="text/javascript">
+    $(window).scroll(function() {
+        var currentScroll = $(window).scrollTop();
+        if (currentScroll >= 268) {
+            $('.sub-nav--hidden').css({
+                'display': 'block',
+                'top': '80px'
```

Comment:
It probably doesn't matter if the hidden version has a `top` of 80px or not.  You should definitely do this in your stylesheet, and the `top` value should be in your `.sub-nav` class.